### PR TITLE
gambit 15.1.1 (new formula)

### PR DESCRIPTION
**Pull request by mcwehner**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/gambit.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  url "https://github.com/gambitproject/gambit/archive/v15.1.1.tar.gz"
+  sha256 "fb4dce2f386e46bbfc72cb75471f43716535937c96ad5a730cad22f97c6a65e6"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "wxmac"
+
+  def install
+    system "aclocal"
```
Comment (DomT4) [inline Formula/gambit.rb]: Should be able to squash this and the next three lines into:
```ruby
system "autoreconf", "-fvi"
```

File: Formula/gambit.rb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    system "glibtoolize"
+    system "automake", "--add-missing"
+    system "autoconf"
+    system "./configure", "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+
+    share.mkpath
```
Comment (DomT4) [inline Formula/gambit.rb]: Redundant when using `.install`.

File: Formula/gambit.rb [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    share.install "contrib"
```
Comment (DomT4) [inline Formula/gambit.rb]: `pkgshare` rather than `share`.

File: Formula/gambit.rb [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  end
+
+  test do
+    system bin/"gambit-enumpure", share/"contrib/games/e02.efg"
```
Comment (DomT4) [inline Formula/gambit.rb]: What are these tests doing, out of curiosity?

Comment (mcwehner) [reply Formula/gambit.rb]: Each test invokes one of Gambit's command-line tools for the given strategic game file, and produces the results on stdout. The intent is really just to make sure the tools can at least run with real (albeit trivial) input and not segfault.

More specifically, `gambit-enumpure` (for example) searches for pure-strategy Nash equilibria for the supplied strategic game. The game referenced by the first test (`e02.cfg`) looks like this:

```
EFG 2 R "Selten (IJGT, 75), Figure 2" { "Player 1" "Player 2" }
""

p "" 1 1 "(1,1)" { "R" "L" } 0
t "" 1 "Outcome 1" { 1, 1 }
p "" 2 1 "(2,1)" { "R" "L" } 0
t "" 2 "Outcome 2" { 0, 2 }
p "" 1 2 "(1,2)" { "r" "l" } 0
t "" 3 "Outcome 3" { 0, 3 }
t "" 4 "Outcome 4" { 2, 0 }
```

This produces the following output, which includes a banner by default:

```
Search for Nash equilibria in pure strategies
Gambit version 15.1.1, Copyright (C) 1994-2014, The Gambit Project
This is free software, distributed under the GNU GPL

NE,1,0,0,0,1,0
```

**Comment by mcwehner** [comment]
Thanks for the feedback: it's helpful, and appreciated.

**Pull request closed without merging**